packages/phone_log/lib/phone_log.dart

```diff
@@ -34,7 +34,7 @@ class PhoneLog {
   /// Request a [permission] and return a [Future] of bool.
   Future<bool> requestPermission() async {
     final bool isGranted =
-        await _channel.invokeMethod("requestPermission", null);
+    await _channel.invokeMethod("requestPermission", null);
```

Comment:
Indent problem.

Reply:
Thanks for catching this. Done.

Reply:
Could you please help merge these changes?  Seems I don't have the access to merge prs to this repo.